### DISCUSSION TITLE
Destroy the camera session

### DISCUSSION THREAD

**Discussion by leahfrom** [enebin/Aespa#52] [Q&A]
Hi, I'm using the camera in an overlay. The problem: When I dismiss the overlay the camera remains active (green dot). I have already tried setting the session to nil but this changes nothing. Calling stopRunning() on the avCaptureSession works but then I have problems spinning the session back up.

Does anyone have a solution for this?

**Comment by ethan-crabb** [comment]
I'm facing the same issue. Did you find a solution?

**Reply by leahfrom** [reply]
Nope..
